packages/yew/src/lib.rs

```diff
@@ -304,6 +306,72 @@ fn set_default_panic_hook() {
     }
 }
 
+#[cfg(feature = "ssr")]
+mod feat_ssr {
+    use super::*;
+
+    use crate::html::Scope;
+
+    /// A Yew Server-side Renderer.
+    #[cfg_attr(documenting, doc(cfg(feature = "ssr")))]
+    #[derive(Debug)]
+    pub struct YewServerRenderer<COMP>
```

Comment:
```suggestion
    pub struct ServerRenderer<COMP>
```
No need to prepend `Yew` to it. `yew::YewServerRenderer` looks weird, compared to `yew::ServerRenderer`

Reply:
There used to be a `YewRenderer` (Removed as that's part of hydration) which is used to supersede the `start_app*` functions (Or else I need to duplicate every function with `hydrate` prefix...).

Do you think `yew::Renderer` is better than `yew::YewRenderer`?

Reply:
> Do you think `yew::Renderer` is better than `yew::YewRenderer`?

Yes. Yew is repeated in `yew::YewRenderer` which makes it worse